frontend/src/views/Footer.js

```diff
@@ -11,6 +11,15 @@ const Footer = () => {
       <div className="build-date">
         Build Date: {process.env.REACT_APP_BUILD_DATE}
       </div>
+      <div className="copyright-disclaimer">
+        <a
```

Comment:
I would have had a text disclaimer instead of a link. But probably worth checking what Brendon actually wants.